src/packet_analysis/protocol/icmp/ICMP.h

```diff
@@ -17,7 +17,7 @@ using RecordValPtr = IntrusivePtr<RecordVal>;
 
 namespace packet_analysis::ICMP {
 
-class ICMPTransportAnalyzer;
+class ICMPAnalyzerAdapter;
```

Comment:
Given that we're in the ICMP namespace now, can skip the ICMP prefix I'd say (similar elsewhere)